app/javascript/services/forest-data.js

```diff
@@ -178,7 +186,7 @@ export const getExtentGrouped = ({
     .replace('{extentYear}', getExtentYear(extentYear))
     .replace('{WHERE}', getWHEREQuery({ iso: adm0, adm1, adm2, ...params }));
 
-  return request.get(url);
+  return axios.get(url, { cancelToken: token });
```

Comment:
what is the cancelToken for?

Reply:
if you try to fetch again for some reason, the first fetch gets cancelled.